config/locales/jobs.yml

```diff
@@ -109,7 +109,7 @@ en:
         other: >-
           within <span class='govuk-!-font-weight-bold'>%{count} %{units}</span> of
       location_html: <span class='govuk-!-font-weight-bold text-capitalize'>&lsquo;%{location}&rsquo;</span>
-      organisation_html: at %{organisation}
+      organisation_html: at <span class='govuk-!-font-weight-bold'>%{organisation}</span>
```

Comment:
anyway html can easily be kept out of translations yml (and below)? not a deal breaker necessarily (the _html postfix is there for a reason but should be avoided if poss imo)